lib/tasks/achievements_import.rake

```diff
@@ -0,0 +1,14 @@
+namespace :desperado do
+
+  desc 'Import achievements into the DB.'
+  task :achievements_import, [:data_file] => :environment do |_, args|
+    File.open("#{Rails.root}/#{args.data_file}", 'r') do |data|
+      achievements = Psych.load_stream data
+      achievements.each do |achievement|
+        Achievement.create achievement
+      end
```

Comment:
Ruby n00b Alert: defining a block that takes one argument and then immediately invoking a method that takes a single argument feels tonk.

I tried passing the `Achievement.create` method as a reference via the `&:` syntax, but Ruby complained at runtime about a missing argument.

If you know a neater way to do this, do school me.